lib/warden/cognito/jwk_loader.rb

```diff
@@ -0,0 +1,28 @@
+module Warden
+  module Cognito
+    class JwkLoader
+      include Cognito::Import['cache']
+      include Cognito::Import['jwk']
+
+      def jwt_issuer
+        jwk.issuer || "https://cognito-idp.#{ENV['AWS_REGION']}.amazonaws.com/#{ENV['AWS_COGNITO_USER_POOL_ID']}"
+      end
+
+      def call(options)
+        return { keys: [jwk.key.export] } if jwk.key.present?
+
+        cache.clear(jwk_url) if options[:invalidate]
```

Comment:
Did you mean `delete`? https://api.rubyonrails.org/classes/ActiveSupport/Cache/Store.html#method-i-delete